### ISSUE TITLE
rename filedescriptors in the kernel

### ISSUE THREAD

**Issue by dgruss** [IAIK/sweb#175]
some students appear to confuse file descriptors with the class of the same name in the kernel. we want to avoid that.

**Comment by Woazboat** [comment]
Good idea. Maybe we could also rename the 'File' class to something like 'FileDescription' (not ideal as the name is quite similar, but that's how it's called by POSIX as well). There has been some confusion between the roles of Inodes and the 'File' class.
